ktor-utils/jvm/src/io/ktor/util/EncodersJvm.kt

```diff
@@ -72,7 +77,7 @@ private fun inflate(
     source: ByteReadChannel,
     gzip: Boolean = true,
     coroutineContext: CoroutineContext
-): ByteReadChannel = GlobalScope.writer(coroutineContext) {
+): ByteReadChannel = GlobalScope.writer(coroutineContext + CoroutineName("encoder-inflate-writer")) {
```

Comment:
Should I create a global field with a pre-created `CoroutineName` to create it only once?